src/rotten-tomatoes/index.js

```diff
@@ -1,3 +1,58 @@
-function rottenTomatoes(grid) {}
+function rottenTomatoes(grid) {
+  let numberOfMinutes = 0;
+
+  // Helper function to check adjacent elements
+  // Turn twos to zeros and ones to twos
+  // Keep track of number of minutes
+  function checkAdjacent(grid, numberOfMinutes) {
+    let queue = [];
+    for (let row = 0; row < grid.length; row++) {
+      for (let col = 0; col < grid[0].length; col++) {
+        if (grid[row][col] === 2) {
+          queue.push([row, col]);
+          grid[row][col] = 0;
```

Comment:
Why do you need to turn them to `0`?

Reply:
I did that so their indexes are ignored in subsequent iterations.